sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -169,6 +169,30 @@ extensions:
                           -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}' \
                           -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_COMMIT )"
     - type: "script"
+      title: "Install service catalog"
+      repository: "aos-cd-jobs"
+      timeout: 1800
+      script: |-
+        if [[ "${ORIGIN_UPGRADE_RELEASE_VERSION}" > 3.6 ]]; then
```

Comment:
better remove this line before we reach 3.10 ... :smile:

Reply:
thanks, updated to use ORIGIN_UPGRADE_RELEASE_MINOR_VERSION instead